samza-core/src/test/scala/org/apache/samza/storage/TestTaskStorageManager.scala

```diff
@@ -711,6 +739,17 @@ class TestTaskStorageManager extends MockitoSugar {
   }
 }
 
+object TestTaskStorageManager {
+
+  @Parameters def parameters: util.Collection[Array[String]] = {
+    val offsetFileNames = new util.ArrayList[Array[String]]()
+    offsetFileNames.add(Array(StorageManagerUtil.OFFSET_FILE_NAME_NEW))
```

Comment:
Is there a test which ensures that if both files exist, then only the NEW one is used?
You might already have a test for this somewhere, but wanted to double check.